lib/project/nativescript-project.ts

```diff
@@ -58,7 +58,7 @@ export class NativeScriptProject extends FrameworkProjectBase implements Project
 	}
 
 	public get startPackageActivity(): string {
-		return StartPackageActivityNames.NATIVESCRIPT;
+		return startPackageActivityNames["nativescript"];
```

Comment:
Since we have `$projectConstants` can we use them here: `return startPackageActivityNames[this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.NativeScript.toLowerCase()];`?